roles/kubernetes-apps/csi_driver/cinder/defaults/main.yml

```diff
@@ -5,6 +5,9 @@
 cinder_auth_url: "{{ lookup('env','OS_AUTH_URL') }}"
 cinder_username: "{{ lookup('env','OS_USERNAME') }}"
 cinder_password: "{{ lookup('env','OS_PASSWORD') }}"
+cinder_application_credential_id: "{{ lookup('env','OS_APPLICATION_CREDENTIAL_ID')  }}"
+cinder_application_credential_name: "{{ lookup('env','OS_APPLICATION_CREDENTIAL_NAME')  }}"
+cinder_application_credential_secret: "{{ lookup('env','OS_APPLICATION_CREDENTIAL_SECRET')  }}"
```

Comment:
The environment variables seem good for me according to https://github.com/kubernetes/cloud-provider-openstack/blob/ab36705fb740299c57cfeeefec6f4185e5f68579/cmd/client-keystone-auth/main.go#L177-L179